tests/util/run.js

```diff
@@ -60,6 +60,18 @@ let nullProxy = new Proxy(
   }
 )
 
+/**
+ * @typedef {object} CrossCheck
+ * @property {typeof import('@jest/globals')} oxide
+ * @property {typeof import('@jest/globals')} stable
+ * @property {object} engine
+ * @property {boolean} engine.oxide
+ * @property {boolean} engine.stable
+ */
+
+/**
+ * @param {(data: CrossCheck) => unknown} fn
```

Comment:
Nitpick:
```suggestion
 * @param {(data: CrossCheck) => void} fn
```
(Is `void` a thing in these docblocks?)

Reply:
It is. This was just my mind picking the wrong word lol

Reply:
Heh no worries!